extern/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif()
 # Add a symlink to USD Composer (create) so that we can use its extensions (e.g. omni.kit.window.material_graph) in our internal applications
 # Don't check for errors because we can still proceed with the build
```

Comment:
We should check for errors, because while the build may still pass if this fails, we can't run the test app.

Reply:
Can we at least log a clear warning to the build output then? This feels like a major footgun.

Reply:
Yeah I can do that

Reply:
Done

![image](https://github.com/CesiumGS/cesium-omniverse/assets/915398/781bc199-3018-4ba4-8a98-0b79a5213e68)